Gemfile

```diff
@@ -102,6 +102,7 @@ end
 group :test do
   gem 'codeclimate-test-reporter', require: false
   gem 'codecov', require: false
+  gem 'database_cleaner'
```

Comment:
Yes! This makes me irrationally exuberant.

Might need to add gemfile.lock changes?

Reply:
Whoops! Yeah, those are now added. Don't know how I missed them.